File: CustomHtmlUi/src/main/java/com/bitmovin/player/samples/custom/ui/html/MainActivity.java
/*
 * Bitmovin Player Android SDK
 * Copyright (C) 2019, Bitmovin GmbH, All Rights Reserved
 *
 * This source code and its use and distribution, is subject to the terms
 * and conditions of the applicable license agreement.
 */

package com.bitmovin.player.samples.custom.ui.html;

import android.content.Intent;
import android.os.Bundle;
import androidx.appcompat.app.AppCompatActivity;
import androidx.cardview.widget.CardView;

public class MainActivity extends AppCompatActivity
{
    @Override
    protected void onCreate(Bundle savedInstanceState)
    {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        CardView cardView = (CardView) this.findViewById(R.id.sintelView);
        cardView.setOnClickListener(v -> startActivity(new Intent(MainActivity.this, PlaybackActivity.class)));
    }
}

File: CustomHtmlUiKotlin/src/main/java/com/bitmovin/player/samples/custom/ui/html/MainActivity.kt
package com.bitmovin.player.samples.custom.ui.html

import android.content.Intent
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.bitmovin.player.samples.custom.ui.html.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {
    private lateinit var binding: ActivityMainBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityMainBinding.inflate(layoutInflater)
        setContentView(binding.root)

        binding.cardView.setOnClickListener {
            startActivity(Intent(this@MainActivity, PlaybackActivity::class.java))
        }
    }
}
